src/main/scala/com/cognite/sdk/scala/v1/assets.scala

```diff
@@ -43,7 +45,8 @@ final case class AssetUpdate(
     metadata: Option[NonNullableSetter[Map[String, String]]] = None,
     parentId: Option[Setter[Long]] = None,
     parentExternalId: Option[Setter[String]] = None,
-    dataSetId: Option[Setter[Long]] = None
+    dataSetId: Option[Setter[Long]] = None,
+    labels: Option[LabelsOnUpdate] = None
```

Comment:
@hakontro This the change for AssetUpdate for labels. It is not a "setter". It has a different structure than the other fields.

I defined and explained it in dataTypes.scala as follows:
```
// Used for updating labels on a data type (only assets for now), examples:
// labels: {add: [{externalId: "label1"}, {externalId: "label2}]}, remove: [{externalId: "label3"}]}
// labels: {add: [{externalId: "label1"}, {externalId: "label2}]}}
final case class LabelsOnUpdate(
    add: Option[Seq[CogniteExternalId]] = None,
    remove: Option[Seq[CogniteExternalId]] = None
)
```